src/main/java/com/google/sps/daos/UserDao.java

```diff
@@ -0,0 +1,5 @@
+package com.google.sps.daos;
+
+public interface UserDao {
+  public void verifyUserInFirestore(String idToken);
```

Comment:
I have seen this type of method usually called `createIfNotExists`

Reply:
Nice, done!